src/world/ObjectMgr.cpp

```diff
@@ -3898,4 +3898,172 @@ void ObjectMgr::LoadAreaTrigger()
     while (result->NextRow());
 
     Log.Success("AreaTrigger", "Loaded %u area trigger teleport definitions", count);
+}
+
+void ObjectMgr::LoadEventScripts()
+{
+    Log.Notice("ObjectMgr", "Loading Event Scripts...");
+
+    QueryResult* result = WorldDatabase.Query("SELECT * FROM event_scripts WHERE event_id > 0 ORDER BY event_id");
```

Comment:
This will not distinguish between a failed query and an empty query. Call the function like this:

bool success = false;
auto result = WorldDatabase.Query(&success, "SELECT * FROM event_scripts WHERE event_id > 0 ORDER BY event_id");

if(!success)
{
    // Error checking code
}
else
{
    if(!result)
    {
        // Table is empty
    }
}